_posts/2024-03-22-release.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Release 2024.12
+author: Priya
+date: 2024-03-22
+category: Jekyll
+layout: post
+---
+### Defect Fixes:
+* Resolved multiple issues in join rule including prefix addition in renamed columns and consistency between foreign and parent views.
```

Comment:
Resolved an issue in join rule due to which columns renamed from foreign View would propagate into the current  View.

Reply:
fixed